stepfunctions/templates/consignment_export_definition.json.tpl

```diff
@@ -5,6 +5,14 @@
     "Run ECS task": {
       "Type": "Task",
       "Resource": "arn:aws:states:::ecs:runTask.sync",
+      "Catch": [
+        {
+          "ErrorEquals": [
+            "States.TaskFailed"
+          ],
+          "Next": "Handle escaped JSON from error cause"
```

Comment:
I know JSON doesn't do comments, but does the template syntax allow them? Can we add one explaining why this step is necessary?

Reply:
https://github.com/hashicorp/terraform/issues/8260 You can't have comments. If you look at a step function execution in the console. you can see why this step is necessary anyway so I don't think this is too bad.